### DISCUSSION TITLE
ECG lead V in the dataset

### DISCUSSION THREAD

**Discussion by lzdh** [MIT-LCP/mimic-code#1460] [MIMIC-General]
Could you kindly let us know which lead is the “V lead” (i.e., V1 to V6)?

**Comment by tompollard** [comment]
Hi @lzdh, my understanding is that the V lead is a chest lead, usually (but not always) V5. I know that you've asked the question before over email, so apologies if I am disagreeing with my past self.  @rgmark or @bemoody may be able to provide a more in-depth explanation!

**Comment by rgmark** [comment]
Much of the data comes from hospital monitors or from ambulatory 
monitors, and the chest electrode is generally placed somewhere in the 
vicinity of V4 - V5. So the precise location of the "V" lead is not 
defined, other than it is the lateral chest wall.

On 10/26/2022 9:52 AM, Tom Pollard wrote:
>
> Hi @lzdh <https://github.com/lzdh>, my understanding is that the V 
> lead is a chest lead, usually (but not always) V5. I know that you've 
> asked the question before over email, so apologies if I am disagreeing 
> with my past self. @rgmark <https://github.com/rgmark> or @bemoody 
> <https://github.com/bemoody> may be able to provide a more in-depth 
> explanation!
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/MIT-LCP/mimic-code/issues/1409#issuecomment-1292085519>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/ABYOOSXDNWVQIIVZWLH7LRTWFEZQZANCNFSM6AAAAAAROYWX7Q>.
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>
>

-- 
Roger G. Mark, M.D., Ph.D.
Distinguished Professor in Health Sciences and Technology, Post-tenure
Institute of Medical Engineering and Science
Room E25-505
MIT
Cambridge, MA 02139
Tel 617-253-7818
Cell 617-359-2429
Fax 617-258-7859

**Comment by tompollard** [comment]
Thanks Roger! @lzdh, hope this helps?

**Comment by lzdh** [comment]
Thanks a lot for the answers!
